h2o-algos/src/main/java/water/rapids/RapidsInit.java

```diff
@@ -0,0 +1,11 @@
+package water.rapids;
+
+import water.rapids.prims.word2vec.AstWord2VecToFrame;
+
+public class RapidsInit {
+
+  public static void registerAlgoRapids() {
+    Env.init(new AstWord2VecToFrame());
```

Comment:
Thank you for the feedback. I made this just to have something quickly.

We can use SPI however we need to keep in mind that the Rapids depend on availability of the algo (which means the corresponding ModelBuilder needs to be registered).

I don't really like the way I implemented this. I think a little cleaner way is to let ModelBuilder to return the rapids (or factory for them) for the particular algorithm.